### DISCUSSION TITLE
Removal of h3_gecko_minversion macro

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#53] [Localization] [answered]
In https://github.com/mdn/content/pull/15086 [teoli2003](https://github.com/teoli2003) removed the last occurrence of the `h3_gecko_minversion` kumascript macro. This macro has now been marked as [deprecated](https://github.com/mdn/yari/pull/5995).

As mentioned in the Yari pull request, there are still [60 occurrences in translated-content](https://github.com/mdn/translated-content/search?q=h3_gecko_minversion). So as you work and encounter this macro, please go ahead and also remove it from the relevant translated content.

Once you have removed all occurrences for a locale, let us know in this discussion (with a link to the relevant pull request) so that we can keep track of the progress. Thank you so much for your continued contribution to MDN Web Docs.

Pinging all translated content leads, so it is on your radar. @mdn/localization-team-leads

**Comment by SphinxKnight** [comment]
Thanks for the notification @schalkneethling, it's really helpful to help "synchronize" such changes. This is tracked on https://github.com/mdn/translated-content/issues/5145 and if I look at a fresh copy of the repo, for active locales, there are a few left for 
- @mdn/yari-content-es (5 files),
- @mdn/yari-content-ja (2 files),
- @mdn/yari-content-pt-br (4 files),
- and @mdn/yari-content-ru (4 files).

`ko`, `zh` and `fr` are green :)

**Comment by Graywolf9** [comment]
Hi! and Thank you so much @schalkneethling @SphinxKnight. I run this and I got 4 files, Am I right?

![Captura de Pantalla 2022-04-28 a la(s) 10 42 34](https://user-images.githubusercontent.com/13079269/165794793-cea70926-0040-4293-b261-78b34d0229be.png)

**Reply by SphinxKnight** [reply]
yes, my bad, there is one `{{TemplateLink("h3_gecko_minversion")}}` which I got as a false positive from the `MDN/Structures/Macros/Other` page

**Reply by Graywolf9** [reply]
Perfect I'm on it, thank you so much!

**Comment by teoli2003** [comment]
Amazing work done until now. We are down to 12 from 60! (`rg -i "\{ *h[123](_gecko)" -l`)

files/pt-br/web/api/console/index.html
files/pt-br/web/api/datatransfer/index.html
files/pt-br/web/api/setinterval/index.html
files/pt-br/web/css/media_queries/using_media_queries/index.html

files/de/web/api/console/index.html

files/ru/web/api/console/index.html
files/ru/web/api/console/group/index.html
files/ru/web/api/datatransfer/index.html
files/ru/web/api/setinterval/index.html

files/ja/web/api/console/group/index.md
files/ja/web/api/setinterval/index.md

files/pl/web/api/console/index.html

**Reply by SphinxKnight** [reply]
https://github.com/mdn/translated-content/pull/5642/files for `pl` and `de`

**Comment by mfuji09** [comment]
I have made PRs to fix:
- https://github.com/mdn/translated-content/pull/5627
- https://github.com/mdn/translated-content/pull/5629

**Comment by SphinxKnight** [comment]
@teoli2003 just had a look at this one, and it seems we're good to go :)

**Reply by teoli2003** [reply]
I confirm I see 0 entries in mdn/translated-content for `h3_gecko_minversion`.

**Comment by teoli2003** [comment, marked as answer]
I think we have successfully removed h3_gecko_minversion from mdn/content and mdn/translated-content \o/

**Reply by Graywolf9** [reply]
Thank you so much @teoli2003 !

**Comment by SphinxKnight** [comment]
@teoli2003 / @schalkneethling can we mark this one as answered?

**Reply by teoli2003** [reply]
I think only the OP can do this. I can't (or it would have been done already)!
